Geomap: Update Documentation (#53552)
grafana/grafana: docs/sources/visualizations/geomap/daynight.md

---
aliases:
  - /docs/grafana/latest/features/panels/geomap/daynight/
  - /docs/grafana/latest/panels/visualizations/geomap/daynight/
description: Night / Day layer
keywords:
  - grafana
  - Geomap
  - panel
  - documentation
title: Night / Day layer
weight: 400
---

# Night / Day

> A layer that displays night and day regions based on current time range.

Reviewer: ```suggestion
The Night / Day layer displays night and day regions based on the current time range.
```

{{< figure src="/static/img/docs/geomap-panel/geomap-day-night-9-1-0.png" max-width="1200px" caption="Geomap panel Night / Day" >}}

## Options

- **Show** toggles time source from panel time range
- **Night region color** picks color for night region
- **Display sun** toggles sun icon
- **Opacity** from 0 (transparent) to 1 (opaque)

{{< figure src="/static/img/docs/geomap-panel/geomap-day-night-options-9-1-0.png" max-width="1200px" caption="Geomap panel Night / Day options" >}}

> ## More Information

Reviewer: ```suggestion
## More information
```
